javascript_yhz/U2/hw4.js

```diff
@@ -0,0 +1,29 @@
+target = 100;
+array1 = [1, 5, 25, 50];
+
+function find2Solutions(array) {
```

Comment:
interesting solution here. i get two questions:
1. have you thought about the complexity? i think it's really hard to compute because of the uncertainty of random
2. there is another requirement which is ONLY 48 coins to be picked.